aatools/food_insecurity/fewsnet.py

```diff
@@ -98,9 +98,9 @@ def download_fewsnet_country(
         f"{base_url_country}?country_code={iso2}"
         f"&collection_date={date.strftime('%Y-%m')}-01"
     )
-    output_dir_country = output_dir / f"{iso2}{date.strftime('%Y%m')}"
-    # question: is there a better way to define this zip path?
-    zip_path_country = Path(f"{output_dir_country}.zip")
+
+    zip_path_country = output_dir / f"{iso2}{date.strftime('%Y%m')}.zip"
```

Comment:
Do we need to save the zip file, or just the output? What do you think of adding to `download_zip` the option of removing the zip file after extraction? Then you could just save the zipfile to `/tmp`